backend/tests/clubs/test_views.py

```diff
@@ -1986,7 +2007,7 @@ def test_club_sensitive_field_renew(self):
         self.client.login(username=self.user4.username, password="test")
 
         with patch("django.conf.settings.QUEUE_OPEN", False):
-            for field in {"name", "description"}:
+            for field in {"name"}:
```

Comment:
Unrelated, but we should have a way to flag description changes up to admin in the future. These descriptions are public -- I can see a world where something inappropriate makes it to a club's description page, and is publicly visible outside of Penn.